features/list_companies_of_category.feature

```diff
@@ -0,0 +1,36 @@
+Feature: As a visitor
```

Comment:
I don't see why we need to test if it is visible to **everyone**, in the agile way that is beyond the scope of this feature. If admin doesn't see it, that is irrelevant for this feature. 
Then the Feature needs to be better described (As `Anyone` or as `Visitor, member and admin`). Right?

I do see why we probably should test 3